The `find_package(root)` command fails when using Root installed via Homebrew.

### Check duplicate issues.

- [ ] Checked for duplicates

### Description

Using root installed via Homebrew in `CMakeLists.txt` fails to locate Vdt.
```
-- The C compiler identification is AppleClang 15.0.0.15000040
-- The CXX compiler identification is AppleClang 15.0.0.15000040
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found nlohmann_json: /usr/local/share/cmake/nlohmann_json/nlohmann_jsonConfig.cmake (found suitable version "3.11.2", minimum required is "3.11.2") 
CMake Error at /usr/local/Cellar/cmake/3.27.8/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Vdt (missing: VDT_INCLUDE_DIR VDT_LIBRARY)
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.27.8/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/Cellar/root/6.30.00/share/root/cmake/modules/FindVdt.cmake:63 (find_package_handle_standard_args)
  /usr/local/Cellar/cmake/3.27.8/share/cmake/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /usr/local/share/root/cmake/rootConfig.cmake:132 (find_dependency)
  CMakeLists.txt:3 (find_package)


-- Configuring incomplete, errors occurred!
```

### Reproducer

- Install Root and CMake using Homebrew.
- Create a simple project.
```cmake
cmake_minimum_required(VERSION 3.20)
project(minimum_root)
find_package(Root CONFIG REQUIRED)
```
- Run CMake with `cmake -S .`

### ROOT version
```
   ------------------------------------------------------------------
  | Welcome to ROOT 6.30/00                        https://root.cern |
  | (c) 1995-2023, The ROOT Team; conception: R. Brun, F. Rademakers |
  | Built for macosx64 on Nov 06 2023, 16:07:43                      |
  | From heads/v6-30-patches@tags/v6-30-00                           |
  | With Apple clang version 15.0.0 (clang-1500.0.40.1)              |
  | Try '.help'/'.?', '.demo', '.license', '.credits', '.quit'/'.q'  |
   ------------------------------------------------------------------
```

### Installation method

homebrew

### Operating system

MacOS(14.1.1)

### Additional context

Since the update in the Homebrew formulae at [this commit](https://github.com/Homebrew/homebrew-core/commit/46e299a429160ea85f93dd15a4359b9cc63e8efb), an issue has arisen. As a temporary workaround, set the following values in `CMakeLists.txt` before `find_package(root)`:
```
set(VDT_INCLUDE_DIR /usr/local/opt/root/include/root)
set(VDT_LIBRARY vdt)
```

--- a/cmake/scripts/ROOTConfig.cmake.in
+++ b/cmake/scripts/ROOTConfig.cmake.in
@@ -136,6 +136,8 @@ if(ROOT_vdt_FOUND AND NOT TARGET VDT::VDT)
       find_dependency(Vdt)
     endfunction()
     find_builtin_vdt()
+  else()
+    find_dependency(Vdt)
   endif()
 endif()
 

